expression/explain.go

```diff
@@ -92,6 +124,20 @@ func SortedExplainExpressionList(exprs []Expression) []byte {
 	return buffer.Bytes()
 }
 
+// SortedExplainNormalizedExpressionList is same like SortedExplainExpressionList, but use for generating normalized information.
+func SortedExplainNormalizedExpressionList(exprs []Expression) []byte {
+	return sortedExplainExpressionList(exprs, true)
+}
+
+// SortedExplainNormalizedScalarFuncList is same like SortedExplainExpressionList, but use for generating normalized information.
+func SortedExplainNormalizedScalarFuncList(exprs []*ScalarFunction) []byte {
+	expressions := make([]Expression, len(exprs))
```

Comment:
Why you need to do a shallow copy here?

Reply:
Because the error: `cannot use exprs (type []*ScalarFunction) as type []Expression in argument to sortedExplainExpressionList`;

Any suggestions?